### YamlMime:UniversalReference
items:
  - uid: azure-servicefabric.ValidationFailedChaosEvent
    name: ValidationFailedChaosEvent
    fullName: ValidationFailedChaosEvent
    children:
      - azure-servicefabric.ValidationFailedChaosEvent.kind
      - azure-servicefabric.ValidationFailedChaosEvent.reason
      - azure-servicefabric.ValidationFailedChaosEvent.timeStampUtc
    langs:
      - typeScript
    type: interface
    summary: 検証中にエラーに対応する混乱するイベントです。
    extends:
      name: azure-servicefabric.ChaosEvent
    package: azure-servicefabric
  - uid: azure-servicefabric.ValidationFailedChaosEvent.kind
    name: kind
    fullName: kind
    children: []
    langs:
      - typeScript
    type: property
    summary: ポリモーフィックな識別子
    syntax:
      content: 'kind: string'
      return:
        type:
          - string
    package: azure-servicefabric
  - uid: azure-servicefabric.ValidationFailedChaosEvent.reason
    name: reason
    fullName: reason
    children: []
    langs:
      - typeScript
    type: property
    summary: ValidationFailedChaosEvent が生成された理由について説明します。 これは、MaxPercentUnhealthyNodes よりは MaxClusterStabilizationTimeout より多くの正常な状態ではないために発生する可能性があります。 この理由は、文字列として、ValidationFailedChaosEvent の理由によりプロパティになります。
    optional: true
    syntax:
      content: 'reason?: string'
      return:
        type:
          - string
    package: azure-servicefabric
  - uid: azure-servicefabric.ValidationFailedChaosEvent.timeStampUtc
    name: timeStampUtc
    fullName: timeStampUtc
    children: []
    langs:
      - typeScript
    type: property
    summary: この混乱イベントが生成されたときの UTC のタイムスタンプ。
    syntax:
      content: 'timeStampUtc: Date'
      return:
        type:
          - Date
    package: azure-servicefabric
references:
  - uid: azure-servicefabric.ChaosEvent
    name: ChaosEvent
    spec.typeScript:
      - name: ChaosEvent
        fullName: ChaosEvent
        uid: azure-servicefabric.ChaosEvent